src/input.c

```diff
@@ -33,7 +33,7 @@ void mouse_button_callback(GLFWwindow* window, int button, int action, int mods)
 void scroll_callback(GLFWwindow *window, double x_offset, double y_offset)
 {
     input *i = (input *) glfwGetWindowUserPointer(window);
-    i->scroll_delta = y_offset;
+    i->scroll_delta = y_offset * -1;
```

Comment:
Don't change the input gathering code, rather change the [world input handling code](https://github.com/knicked/CCraft/blob/f6c98ebec43173c0b8f4e48eba798d6b21059372/src/world.c#L209).